### ISSUE TITLE
Gulp Slow

### ISSUE THREAD

**Issue by solepixel** [RelevanceDigital/wp-bs-starter-theme#4]
I'm not sure if it has to do with my local setup or the Gulp configuration, but using the default `gulp` command on projects can sometimes take 5-10s for SCSS to compile and another 5-10s to be injected into the browser. This slows down production substantially and is quite frustrating.

Would there be any problems switching to something like [WPGulp](https://github.com/ahmadawais/WPGulp)? I know it hasn't been updated in a while, but could be forked and maintained. I have used this on other themes/plugins and it works much better in my experience.

**Comment by robwent** [comment]
Yes, it seemed to slow down recently, I think after some updates for recent gulp versions.

I don't work for relevance any more so it would be up to them but I don't see any issue using another gulp file.

You could try removing js and image compression on the last line to see if that speeds it up.
